### DISCUSSION TITLE
Cant show page content and livechart at the same time. Maui 2. NET 8

### DISCUSSION THREAD

**Discussion by Malher120** [Live-Charts/LiveCharts2#1593] [Q&A]
Im trying to show my livechart together with an already existing maui app but the chart will remove all other content when I try to put it in. If I try to add it inside the vertical or horizontal stacks, the chart is invisible 
`<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="AlkMauiApp.ResultPage"
             xmlns:lvc="clr-namespace:LiveChartsCore.SkiaSharpView.Maui;assembly=LiveChartsCore.SkiaSharpView.Maui"
             xmlns:local="clr-namespace:AlkMauiApp.ViewModels;assembly=AlkMauiApp"
             Title="ResultPage">
    <ContentPage.BindingContext>
        <local:ViewModel/>
    </ContentPage.BindingContext>
    <VerticalStackLayout>
        <HorizontalStackLayout>
             <VerticalStackLayout
    Margin="25"
    VerticalOptions="Start"
    HorizontalOptions="Center"
    >
                    <Label
        FontSize="Header"
        Text="Indata:"
        VerticalOptions="Start">

                    </Label>
                    <Label
        x:Name="input1Label">
                    </Label>

                    <Label
        x:Name="input2Label">
                    </Label>

                    <Label
        x:Name="input3Label">
                    </Label>
                </VerticalStackLayout>

                <VerticalStackLayout
    Margin="25"
    VerticalOptions="Start"
    HorizontalOptions="Center">
                    <Label
        FontSize="Header"
        Text="Utdata:">

                    </Label>
                    <Label 
        x:Name="alkalinityLabel">

                    </Label>
                    <Label 
        x:Name="phLabel">

                    </Label>
                    <Label
        x:Name="totalLabel">
                    </Label>
                    <Label
        x:Name="poeLabel">
                    </Label>
                    <Label
        x:Name="co2Label">
                    </Label>
                    <Label
        x:Name="hco3minLabel">
                    </Label>
                    <Label
        x:Name="co32minLabel">
                    </Label>
                    <Label
        x:Name="ohminLabel">
                    </Label>
                    <Label
        x:Name="ca2plusLabel">
                    </Label>

                </VerticalStackLayout>
            </HorizontalStackLayout>
        </VerticalStackLayout>
    
</ContentPage>`

**Comment by beto-rodriguez** [comment]
Hello.

I will move this to the Q&A section, maybe someone can help you there, I am afraid that this issue is related on to your app layout, with more context I will be able to help you.
